webpack/development.config.js

```diff
@@ -1,19 +1,17 @@
 const webpack = require('webpack');
+const merge = require('webpack-merge');
```

Comment:
Is it really necessary to include this extra dependency? My point is that it’s not worth it because current merge logic isn’t really complicated and is very straightforward. Including `webpack-merge` will require it’s API knowledge, and this is another dependency that increases `npm install` time and complicates the dependencies update process.

Reply:
1) It has a simple API
2) It's small - https://bundlephobia.com/result?p=webpack-merge@4.1.2
3) It's comfortable. If we make changes to `common.config.js`, then we don't have to write anything else.
4) A line similar to `...baseWebpackConfig` makes reading the configuration file difficult.

I insist on using the `webpack-merge`, but you make the final decision.